components/camel-sql/src/main/java/org/apache/camel/component/sql/stored/SqlStoredEndpoint.java

```diff
@@ -59,6 +59,9 @@
     @UriParam(description = "Whether this call is for a function.")
     private boolean function;
 
+    @UriParam(label = "template")
```

Comment:
the label in the uriParam is related to generic camel labels, template is not one of them

Reply:
Okay , I thought since this option would be set on template, i preassumed it should be a template. Let me correct it,